add-dynamic-entity-layer/src/main/java/com/esri/arcgismaps/sample/adddynamicentitylayer/components/BottomAppContent.kt

```diff
@@ -0,0 +1,196 @@
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.adddynamicentitylayer.components
+
+import android.content.res.Configuration
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleTypography
+
+@Composable
+fun DynamicEntityLayerProperties(
+    isTrackLineVisible: (Boolean) -> Unit = { },
+    arePrevObservationsVisible: (Boolean) -> Unit = { },
+    onObservationsChanged: (Float) -> Unit = { },
+    purgeAllObservations: () -> Unit = { },
+    trackLineCheckedState: Boolean,
+    prevObservationCheckedState: Boolean,
+    trackSliderValue: Float,
+    onDismiss: () -> Unit = { }
+) {
+    var sliderValue by remember { mutableStateOf(trackSliderValue) }
+    // set the state of the switch
+    var trackLineState by remember { mutableStateOf(trackLineCheckedState) }
+    var prevObservationState by remember { mutableStateOf(prevObservationCheckedState) }
+
+    Column(
+        Modifier.background(MaterialTheme.colorScheme.background)
+    ) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(
+                modifier = Modifier.weight(5f),
+                color = MaterialTheme.colorScheme.primary,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center,
+                text = "Dynamic Entity Settings",
+            )
+            TextButton(
+                modifier = Modifier.weight(1f),
+                onClick = onDismiss
+            ) {
+                Text(text = "Done")
+            }
+        }
```

Comment:
Good Suggestions, thank you.

In this case, the `Done` TextButton instead of a close (X) IconButton to dismiss the sheet, would be preferable.
As the bottomsheet is an editable settings page. The user would apply the settings to the DEL, and when they are done applying, they click `Done`. 
The X would make more sense, if the bottomsheet was only readable content (show results of an identify result/ popup result) instead of editable content.